libs/database/src/lib/schemas/user.schema.ts

```diff
@@ -271,6 +272,10 @@ UserSchema.methods.toPageResponse = function (
     blocked,
     blocking,
     followed,
+    socialSyncs:
+      (this as User).profile && (this as User).profile.socialSyncs
+        ? (this as User).profile.socialSyncs
+        : null,
```

Comment:
```suggestion
    socialSyncs: (this as User).profile?.socialSyncs ?? null,
```

[REF](https://medium.com/@suraneti.rod/javascript-optional-chaining-%E0%B8%84%E0%B8%B7%E0%B8%AD%E0%B8%AD%E0%B8%B0%E0%B9%84%E0%B8%A3-%E0%B9%83%E0%B8%99-es11-%E0%B8%97%E0%B8%B5%E0%B9%88%E0%B8%81%E0%B8%B3%E0%B8%A5%E0%B8%B1%E0%B8%87%E0%B8%88%E0%B8%B0%E0%B8%A1%E0%B8%B2%E0%B8%96%E0%B8%B6%E0%B8%87-b3bec898ec76)